tableau_utilities/general/config_column_persona.py

```diff
@@ -0,0 +1,72 @@
+personas = [
```

Comment:
@JustinGrilli  I've moved this into the package. For Airflow I think it's better if this is imported from the package than for users to have to copy and paste the file in every case.  This will keep it standardized better.

Reply:
Sounds good. My only request is that we make this a yaml file :)

Reply:
@JustinGrilli  what's the reasoning for that? That seems to me like to makes other code more complex.

 It's currently the proper data type to import into other code by just writing `import ...` without having to do any type conversions.  Making it a `.yaml` file will require extra code everywhere it's used to read the file and make sure the data type is right.  I think it's way easier to use to make it a Python data type from the start.

Reply:
Yeah its fine to leave it this way. I just personally like working with yaml files. Since I highly doubt we will iterate much if at all with this file tho, yaml really isn't necessary. I'm fine to leave it as is.